lib/tasks/deliver.rake

```diff
@@ -1,13 +1,19 @@
 namespace :deliver do
+  def test_email
+    OpenStruct.new(subject: "Test email", body: "This is a test email,")
```

Comment:
I think there should be a `.` at the end of the string, rather than a `,`!

Reply:
quite right....good spot.